frontend/packages/kubevirt-plugin/src/components/vm-status/vm-statuses.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import { PodKind, K8sResourceKind } from '@console/internal/module/k8s';
+import { getNamespace, getName } from '@console/shared';
+import { VM_STATUS_IMPORTING, VM_STATUS_IMPORT_ERROR } from '../../statuses/vm/constants';
+import { VMKind } from '../../types';
+import { getVMStatus } from '../../statuses/vm/vm';
+import { getVMImporterPods } from '../../selectors/pod/selectors';
+import { VmStatus } from './vm-status';
+
+const getId = (value) => `${getNamespace(value)}-${getName(value)}`;
+
+export const VmStatuses: React.FC<VmStatusesProps> = (props) => {
+  const { vm, pods, migrations } = props;
+  const statusDetail = getVMStatus(vm, pods, migrations);
+  const importerPods = getVMImporterPods(pods, vm);
+
+  switch (statusDetail.status) {
+    case VM_STATUS_IMPORTING:
+    case VM_STATUS_IMPORT_ERROR:
+      return (
+        <>
+          {importerPods.map((pod) => (
+            <div key={getId(pod)}>
```

Comment:
Do we really need the `<div>` wrapper?

If not, you can simply declare the `key` prop on `<VMStatus>` below.

Reply:
The `VMStatus` can (rarely) render `React.Fragment` with text. So this `div` ensures the statuses to be shown one per line.

Reply:
we could use `pod.metadata.uid` instead
```suggestion
            <div key={pod.metadata.uid}>
```

Reply:
Why? The reason for `getId` helper is to have a common way to access `metadata.uid` on a resource.

Reply:
this is using `getId` https://github.com/openshift/console/pull/2924/files#diff-c4803c82c2975654bafdd7fdc6c394a7R10 not `getUID` selector. Im proposing to use pod's `uid` as `key` and sure, we can use `getUID` selector for that

Reply:
I see, thanks for clarifying @rawagner